lib/pure/md5.nim

```diff
@@ -237,29 +271,41 @@ proc md5Init*(c: var MD5Context) =
   c.state[3] = 0x10325476'u32
   c.count[0] = 0'u32
   c.count[1] = 0'u32
-  zeroMem(addr(c.buffer), sizeof(MD5Buffer))
+  when false:
```

Comment:
The `when false`s are because I am not sure if it would be better to just remove the `zeroMem`/`copyMem` versions than special casing them for non-VM non-JS.